index.js

```diff
@@ -141,7 +141,7 @@ module.exports = function (log, isReady, mapper) {
               Looper,
               sv.createSink(function (err) {
                 if (!flume.closed) {
-                  if (err) {
+                  if (err && err.abort !== true) {
```

Comment:
I'd like a comment somewhere in here which says: 

```
// Interpreting different `err` values:
// - null : this is a happy normal closure, let's happily close down
// - { Error } : a genuine problem was hit, we need to fix the view
// - { Error, abort: true } : the stream was deliberately aborted, ...   
```

see I can't finish these accurately. I think while it's fresh it'd be good to write things down?

Reply:
That's correct! I'll make a note too.